### DISCUSSION TITLE
How to change html tags used text node | <s></s>

### DISCUSSION THREAD

**Discussion by topstartup** [facebook/lexical#4667] [General]
Is there any way to change the tags used for text formatting? For example, underline and strikethrough using `<u></u>` and `<s></s>`. I would change the strikethrough to `<del></del>`.

` <p><u><span>text</span></u></p><p><s><span>text</span></s></p>`

Basically, when I dispatch text command with 'strikethrough', instead of the default implementation of text node using the tag `<s>`, I want to use `<del>` tag.

**Comment by moy2010** [comment]
You will need to override the `LexicalTextNode`. See the source file for inspiration: https://github.com/facebook/lexical/blob/main/packages/lexical/src/nodes/LexicalTextNode.ts
